pkg/oauth/oauth.go

```diff
@@ -291,6 +292,28 @@ func (s *server) Token(w http.ResponseWriter, r *http.Request) {
 			ar.Authorized = true
 		}
 	}
+	if client.TieAccessToSession {
+		session, err := s.store.GetSession(
+			r.Context(),
+			&ttnpb.UserIdentifiers{UserId: userIDs.UserId},
+			ar.UserData.(userData).UserSessionIdentifiers.GetSessionId(),
+		)
+		if err != nil {
+			webhandlers.Error(w, r, err)
+			return
+		}
+		if session == nil {
+			ar.Authorized = false
+		} else {
+			secondsUntilExpiry := int32(ttnpb.StdTime(session.ExpiresAt).Sub(time.Now()).Seconds())
```

Comment:
We prefer to use `time.Duration` instead of "seconds".

```suggestion
			timeUntilExpiry := time.Until(ttnpb.StdTime(session.ExpiresAt))
			// etc.
```